app/templates/create.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
+  
+  <form class="container my-1 rounded bg-white border border-primary" id="createForm" data-link="{{ url_for('create') }}">
+    {{ form.hidden_tag() }}
+
+    <div class="row my-3 gy-2">
+      {{ form.title.label(class="col-form-label col-2") }}
+      <div class="col-10">
+        {{ form.title(class="form-control") }}
```

Comment:
If I understand this correctly, submitting an invalid form will refresh the page and load in the errors when it refreshes?

Reply:
Yes, It should to me understanding and research (if youre curious re-read the server side flask lecture, and do part 3 of the flask mega tutorial, where this is covered:) )